modules/mobile/app/controllers/mobile/v0/prescriptions_controller.rb

```diff
@@ -11,6 +11,8 @@ def index
         begin
           resource = client.get_history_rxs
 
+          # this was recently commended out because it was suspected of causing performance issues due to the nested logging
```

Comment:
Removed all of this logging. It was all for debugging. The rescues that capture errors are theoretically useful but they mostly capture expected errors. It would be great if we could distinguish between useful and useless errors, but until then we shouldn't be doing this unless we have a specific problem we're trying to debug.